mico-core/src/main/java/io/github/ust/mico/core/web/ServiceInterfaceController.java

```diff
@@ -239,7 +240,7 @@ private MicoService getServiceFromDatabase(String shortName, String version) thr
      * @param service
      * @return
      */
-    private boolean serviceInterfaceExists(@RequestBody MicoServiceInterface serviceInterface, MicoService service) {
```

Comment:
Are you sure to remove @RequestBody?

Reply:
The annotation is not necessary there because serviceInterfaceExists does not handle requests.